llama_index/query_engine/sub_question_query_engine.py

```diff
@@ -156,8 +159,22 @@ async def _aquery_subq(
             if self._verbose:
                 print_text(f"[{sub_q.tool_name}] Q: {question}\n", color=color)
 
+            event_start_payload = {EventPayload.QUERY_STR.value: question}
```

Comment:
Correct me if I'm wrong, but I don't think the `.value` is needed here (it wasn't used on the other payloads)

Reply:
Hmm yeah I was getting type check warnings without using `.value` 🤔 I'll double check on that

Reply:
Yeah it was because of [these errors from the lint check](https://github.com/jerryjliu/llama_index/actions/runs/5468178774/jobs/9955462458?pr=6716). Surprised that it didn't error for the other lines where we're not using `.value` 🤔